src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -82,6 +82,24 @@ class Response
     const HTTP_NOT_EXTENDED = 510;                                                // RFC2774
     const HTTP_NETWORK_AUTHENTICATION_REQUIRED = 511;                             // RFC6585
 
+    const HTTP_RESPONSE_CACHE_CONTROL_DIRECTIVES = [
```

Comment:
`// see developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control` on top would be helpful

Reply:
P.S : `last-modified` and `etag` are not http cache control directives but they are used in `setCache`. should we remove them from this array ? also, should we add `stale-while-revalidate` and `stale-if-error` ( extensions, not part of core )

Reply:
note: if we remove `last-modified` and `etag` we would still need to support both in `setCache` otherwise that would be considered a BC-break. but maybe we can deprecate setting these two via `setCache` and suggest using `setLastModified` and `setETag` in the future instead ?

Reply:
no need to change something that works